README.md

```diff
@@ -100,6 +139,14 @@ final class AppDelegate: ApplicationDelegate {
 	where `libpython3.10.so` is currently installed version and libpython3.8.so.1.0 is filename against which Swifts LLDB was built.
   </details>
 After the necessary dependencies were set up just add this package in your SwiftPM manifest file as a dependency and add **AppKid** product as a dependency to your target:
+
+Eg:
+```bash
+ mkdir MyApp
+ cd MyApp
+ swift package init --type executable
+```
+Then replace the generated Package.swift in this folder with something like the sample below:
```

Comment:
I've added a big comment on this on our original conversation. I still believe this piece does not belong here.